src/main/java/soconnect/ui/CommandBox.java

```diff
@@ -72,57 +80,89 @@ private void handleCommandEntered() {
     }
 
     /**
-     * Displays a list of autocomplete entries.
-     * Solution below adapted from https://stackoverflow.com/questions/36861056/javafx-textfield-auto-suggestions.
+     * A Listener to execute autocomplete related actions.
```

Comment:
I'm afraid the convention forces us to start with active verb :")

```suggestion
     * Activates a Listener to execute autocomplete related actions.
```